src/test/ui/float-literals.rs

```diff
@@ -0,0 +1,12 @@
+// run-pass
```

Comment:
Can you put this in a subdirectory? src/test/ui is already giant 😅

Reply:
I moved it to `consts`, since there really isn't anything closer to this. I get it really doesn't make sense, but 🤷.

Reply:
`consts` is specifically about CTFE (`const` items, `const fn`). Please don't put unrelated tests there. :)

Does `ui/parser` make sense?

Reply:
Yeah, probably a lot more sense than `ui/consts`. I'll move it there then.

Reply:
[Moved](https://github.com/Alexhuszagh/rust/blob/fb1a817fa13f774ce4da27fb9769db2779163030/src/test/ui/parser/float-literals.rs) `src/test/ui/float-literals.rs` to `src/test/ui/parser/float-literals.rs`.